apps/dapp/src/components/Pages/Ascend/components/LBPForm/index.tsx

```diff
@@ -119,73 +86,133 @@ export const LBPForm = ({ pool }: Props) => {
   };
 
   const createChangeHandler = (fieldKey: keyof Values) => (event: React.ChangeEvent<any>) => {
-    setFormValue(fieldKey, event.target.value);
+    let value = event.target.value;
+
+    if (['startWeight1', 'startWeight2', 'endWeight1', 'endWeight2'].indexOf(fieldKey) >= 0) {
+      value = DecimalBigNumber.parseUnits(value, 16);
```

Comment:
there are 16 decimals on these numbers? not 18?

Reply:
The issue is the value from the input is already taking up 2 decimal places, e.g. 80 vs 0.8. So we get two decimal places "for free".